docs/guides/unbound.md

```diff
@@ -52,82 +52,154 @@ The first thing you need to do is to install the recursive DNS resolver:
 sudo apt install unbound
 ```
 
-**Important**: Download the current root hints file (the list of primary root servers which are serving the domain "." - the root domain). Update it roughly every six months. Note that this file changes infrequently.
-```
-wget -O root.hints https://www.internic.net/domain/named.root
-sudo mv root.hints /var/lib/unbound/
-```
-
 ### Configure `unbound`
 Highlights:
-- Listen only for queries from the local Pi-hole installation (on port 5353)
+- Listen only for queries from the local Pi-hole installation (on port 10053)
 - Listen for both UDP and TCP requests
 - Verify DNSSEC signatures, discarding BOGUS domains
 - Apply a few security and privacy tricks
 
  `/etc/unbound/unbound.conf.d/pi-hole.conf`:
 ```ini
 server:
+    # One thread should be sufficient, can be increased on beefy machines. In reality for most users running on small networks or on a single machine it should be unnecessary to seek performance enhancement by increasing num-threads above 1.
+    num-threads: 1
+
     # If no logfile is specified, syslog is used
     # logfile: "/var/log/unbound/unbound.log"
     verbosity: 0
 
-    port: 5353
+    # Ensure Unbound is running on loopback interface
+    interface: 127.0.0.1@53
```

Comment:
What is the default if you do not specify this line?

Reply:
The default when this line is omitted is to bind to localhost, but I find that clearly showing which interface would prevent people using 0.0.0.0 on their own. Using the @port method makes the "port:" option redundant too. Maybe one or the other?